releng/org.eclipse.xtext.sdk.feature/feature.xml

```diff
@@ -67,10 +67,10 @@ SPDX-License-Identifier: EPL-2.0
          version="0.0.0"/>
 
    <requires>
-      <import plugin="org.eclipse.emf.ecore" version="2.20.0" match="greaterOrEqual"/>
-      <import plugin="org.eclipse.emf.common" version="2.17.0" match="greaterOrEqual"/>
-      <import plugin="org.eclipse.emf.ecore.source" version="2.20.0" match="greaterOrEqual"/>
-      <import plugin="org.eclipse.emf.common.source" version="2.17.0" match="greaterOrEqual"/>
+      <import plugin="org.eclipse.emf.ecore" version="2.26.0" match="greaterOrEqual"/>
```

Comment:
Gosh the way we manage dependencies is mildly confusing to say the least.
The xtext sdk certainly would need the EMF generator along with the ecore itself.

Anyhow, not actionable. Approved